src/PrestaShopBundle/Resources/config/services/core/util/helper_card.yml

```
@@ -71,3 +71,9 @@ services:
           'es': 'https://doc.prestashop.com/display/PS17/Facturas+por+abono'
           'it': 'https://doc.prestashop.com/display/PS17/Note+di+credito'
           '_fallback': 'https://doc.prestashop.com/display/PS17/Credit+Slips'
+        'carrier':
+          'en': 'https://doc.prestashop.com/display/PS17/Carriers'
+          'fr': 'https://doc.prestashop.com/pages/viewpage.action?pageId=51840218'
```

Waho 😮 there should be a `https://doc.prestashop.com/display/PS17/Gerer+les+transporteurs` but it does not exist and the link you use is right.

@Julievrz @LouiseBonnard please see this issue. We have a weird page for "Managing carriers" French page on doc.prestashop.com 🤔

---

I know... it happens because Confluence does not support accented letters. So we have the possibility to:

- keep the accented character in the French title of the page, and so the link
- remove the accent from the title and therefore get a more friendly link

Knowing that it will probably be fixed when the user documentation will be officially hosted on GitBook.

---

@LouiseBonnard what should we do about this problem since it should not be the case once on GitBook?

---

I've just removed the accent from the title and updated the user documentation cartography ([here](https://docs.google.com/spreadsheets/d/13VhQNdYR9-89KBBbGNRTIk9HSXrjzxTXl52AoHswI2E/edit#gid=0)).

```suggestion
          'fr': 'https://doc.prestashop.com/display/PS17/Gerer+les+transporteurs'
```

---

Thank you @LouiseBonnard !

---

Hi @ks129, don't forget this suggestion please! ☺️ Thank you